src/course-outline/CourseOutline.test.jsx

```diff
@@ -483,6 +485,93 @@ describe('<CourseOutline />', () => {
     expect(datePicker).toHaveValue('08/10/2025');
   });
 
+  it('check configure subsection when configure subsection query is successful', async () => {
+    const {
+      findAllByTestId,
+      findByText,
+      findAllByPlaceholderText,
+      getByText,
+      getByRole,
+      getAllByRole,
+      getByTestId,
```

Comment:
We don't need to have both `find*` and `get*` functions. You can replace all `get*` calls by `await find*` calls.